### ISSUE TITLE
Gradle 6 compatibility

### ISSUE THREAD

**Issue by altavir** [Kotlin/kotlinx-benchmark#15]
Currently using with gradle 6 gives:
```
Execution failed for task ':examples:benchmarksBenchmarkGenerate'.
> There was a failure while executing work items
   > A failure occurred while executing kotlinx.benchmark.gradle.JmhBytecodeGeneratorWorker
      > Generation of JMH bytecode failed with 1errors:
          - Group name should be the legal Java identifier.
           [org.openjdk.jmh.generators.reflection.RFMethodInfo@3a40cc4e]
```

**Comment by CommanderTvis** [comment]
Reproduced with Gradle 6.5. 
```java
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':examples:benchmarksBenchmarkGenerate'.
> There was a failure while executing work items
   > A failure occurred while executing kotlinx.benchmark.gradle.JmhBytecodeGeneratorWorker
      > Generation of JMH bytecode failed with 1errors:
          - Group name should be the legal Java identifier.
           [org.openjdk.jmh.generators.reflection.RFMethodInfo@66de225]
```

**Comment by altavir** [comment]
The problem comes from quoted benchmark function names with spaces.

**Comment by qurbonzoda** [comment]
This problem is fixed with version `0.2.0-dev-20`.
Thanks for your feedback!
